docs/whats-new.md

```diff
@@ -1,5 +1,15 @@
 # What's New
 
+## v6.0
+
+Release date: TBD
+
+* NEW: `getProjectionMatrix` can match the behavior of the latest Mapbox (when called witch `farZMultiplier: viewport.height`)
+* NEW: `addMetersToLngLat` utility
+* FIX: returned values by `getDistanceScales` now reflect the flipped y-direction between spherical coordinates and the WebMercator world.
+* FIX: `WebMercatorViewport`'s `project` and `unproject` handle altitude correctly
+* BREAKING: `getWorldPosition` is removed
```

Comment:
Nit: Should we add an upgrade guide? Or at least say something about how this can be worked around/implications.

Reply:
I honestly do not think anyone besides deck.gl was using it...